db/migrate/20181002192054_fix_conversion_host_resource_type.rb

```diff
@@ -0,0 +1,13 @@
+class FixConversionHostResourceType < ActiveRecord::Migration[5.0]
+  class ConversionHost < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Converting invalid resource_type to Host") do
+      ConversionHost.in_my_region
```

Comment:
I think this should only be scoped if the other migration is scoped.

Reply:
Want to change the other migration to be scoped in this PR?

Reply:
Yes, please